vizro-core/src/vizro/models/_components/table.py

```diff
@@ -0,0 +1,55 @@
+import logging
+from typing import List, Literal
+
+import pandas as pd
+from dash import dash_table, html
+from pydantic import Field, PrivateAttr, validator
+
+import vizro.tables as vt
+from vizro.managers import data_manager
+from vizro.models import Action, VizroBaseModel
+from vizro.models._action._actions_chain import _action_validator_factory
+from vizro.models._components._components_utils import _process_callable_data_frame
+from vizro.models._models_utils import _log_call
+from vizro.models.types import CapturedCallable
+
+logger = logging.getLogger(__name__)
+
+
+class Table(VizroBaseModel):
+    """Wrapper for table components to visualize in dashboard.
+
+    Args:
+        type (Literal["table"]): Defaults to `"table"`.
+        figure (CapturedCallable): Table like object to be displayed. Current choices include:
+            [`dash_table.DataTable`](https://dash.plotly.com/datatable).
+        actions (List[Action]): See [`Action`][vizro.models.Action]. Defaults to `[]`.
```

Comment:
We need to expose a `title` argument - distinct to the Graph model the dash datatable doesn't have a native title argument embedded. So we would probably have to expose it here and in the return it should be something like below: 

`return html.Div([html.H3(self.title) if self.title else None, dash_table.DataTable(pd.DataFrame().to_dict("records")], []), className="table-container", id=self.id)`

I think it's H3, but double-check, it should be of the same size as the chart title in Graph